tests/integration/src/test/java/org/apache/pulsar/tests/integration/schema/SchemaTest.java

```diff
@@ -130,38 +131,33 @@ public void testMultiVersionSchema() throws Exception {
                 tenant + "/" + namespace,
                 Sets.newHashSet(pulsarCluster.getClusterName())
         );
-
-        // Create a topic with `Person`
-        try (Producer<Person> producer = client.newProducer(Schema.AVRO(
+        Producer<Person> producer = client.newProducer(Schema.AVRO(
```

Comment:
any reason why do you remove `try` here?

Reply:
we need to produce message after consumer subscribe and send should outside the try block, so we can't use try but we need to close the producer and consumer when end the test.